app/src/main/java/org/fossasia/susi/ai/activities/ImportantMessages.java

```diff
@@ -94,4 +98,16 @@ public void run() {
             }
         });
     }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem menuItem)
+    {
+        switch (menuItem.getItemId()) {
+            case android.R.id.home:
+                onBackPressed();
```

Comment:
Not the best way to implement this imo.
It's better if you use the NavUtils class for providing up naivgation.
https://developer.android.com/training/implementing-navigation/ancestral.html#NavigateUp